### PR TITLE
Auto update

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: etc/fields.yml [hunk 1]
```diff
@@ -13,10 +13,18 @@ env:
       These fields contain data about the environment in which the
       transaction was captured.
   fields:
-    - name: shipper
+    - name: beat.hostname
```
Comment (ruflin) [inline etc/fields.yml]: @tsg Do you know why the packetbeat system tests running before without these changes?

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+os:
+  - linux
+  - osx
+
 go:
   - 1.5.1
 
+env:
+  - TARGETS="check testsuite"
+#  - TARGETS="crosscompile"
```
Comment (urso) [inline .travis.yml]: why no more crosscompile?

Comment (ruflin) [inline .travis.yml]: Doesn't work because of cgo

Comment (urso) [inline .travis.yml]: is there a way to re-enable later? We want to create an issue?

Comment (ruflin) [inline .travis.yml]: @andrewkroh knows more here

**Comment by urso** [comment]
big change, but LGTM

File: etc/fields.yml (hunk 1 above)
Comment (tsg) [inline etc/fields.yml]: Because libbeat was not yet updated (the PR that doesn't isn't merged).

File: .gitignore [hunk 3]
```diff
@@ -10,7 +10,6 @@
 packetbeat.test
 *.dev.yml
```
Comment (tsg) [inline .gitignore]: For my understanding, is `.gitignore` simply copied over from libbeat? We should maybe put a warning comment not to change the ones from the Beats directly.

File: Makefile [hunk 4]
```diff
@@ -1,31 +1,15 @@
-GODEP=$(GOPATH)/bin/godep
+#!/bin/bash
```
Comment (tsg) [inline Makefile]: hashbang in a Makefile? What does that mean?

File: .gitignore (hunk 3 above)
Comment (ruflin) [inline .gitignore]: No, gitignore is not copied over.

File: Makefile (hunk 4 above)
Comment (ruflin) [inline Makefile]: Yep, that is not needed here :-( Will remove it.

File: etc/packetbeat.yml [hunk 5]
```diff
@@ -156,13 +159,16 @@ output:
     # IPv6 addresses should always be defined as: https://[2001:db8::1]:9200
     hosts: ["localhost:9200"]
 
-    # Optional protocol and basic auth credentials. These are deprectad
+    # Optional protocol and basic auth credentials. These are deprectated.
```
Comment (tsg) [inline etc/packetbeat.yml]: This typo just comes from libbeat, right? So we have to fix it there and then call update again. Do I get it right?

File: Makefile (hunk 4 above)
Comment (ruflin) [inline Makefile]: @tsg It is also in the Makefile from libbeat. Will remove it there too which will then remove it with the next update from all repos.

File: etc/packetbeat.yml (hunk 5 above)
Comment (ruflin) [inline etc/packetbeat.yml]: yes. Will do it.

File: .gitignore (hunk 3 above)
Comment (tsg) [inline .gitignore]: Cool, I think that's better.

File: .travis.yml (hunk 2 above)
Comment (andrewkroh) [inline .travis.yml]: We never had that capability before for Packetbeat. `gopacket` uses cgo so we would have to have the gcc toolchain available for each os/arch.

**Pull request merged**